cfme/base/ui.py

```diff
@@ -1077,6 +1079,20 @@ def step(self):
         self.prerequisite_view.help_menu.select()
 
 
+@navigator.register(Region)
```

Comment:
Optional:
But, I like to go with registration name like L1357 you added. rather than taking `class` name.

Reply:
I'd keep it like that, otherwise I need to re-write navigation registration for other tabs in `Region`. Now they are in the same pattern.

Reply:
oh no, I meant I need to re-write it to be written in the same way, not because it will be broken :)